site.py

```diff
@@ -315,11 +320,13 @@ def check_container(cid):
 # Updated using the function above.
 @app.route('/containers/<uuid>/status', methods=['GET'])
 def container_status(uuid):
-    cid = container_id(uuid)
-    url = '/check_container/' + cid[:16]
+    url = launch_container(uuid)
```

Comment:
The check_container() method already returns a status 'Missing' if the container is not found.